### PR TITLE
Add degrees

### LINKED ISSUE #207: Transition alignment per technology, missing degree symbol in axes
Low priority

Screen Shot 2022-11-22 at 8 12 32 PM

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #207

### CODE REVIEW

File: R/lookup.R [hunk 1]
```diff
@@ -16,4 +16,4 @@ index_cb_short_lookup <- "iShares_Global_Corp_Bond"
 
 scatter_sector_lookup <- "Power"
 
-alignment_table_temperatures_lookup <- c("<1.5C", "1.5-1.8C", ">1.8C")
+alignment_table_temperatures_lookup <- c("<1.5\u00B0C", "1.5-1.8\u00B0C", ">1.8\u00B0C")
```
Comment (jdhoffa) [inline R/lookup.R:19]: does there need to be a space after the escaped string? like `"<1.5\u00B0 C"`

Comment (jdhoffa) [reply R/lookup.R:19]: otherwise won't the C be interpreted as part of the escape sequence

**Review by cjyetman — APPROVED** [review]

**Review by jacobvjk — APPROVED** [review]

File: R/lookup.R (hunk 1 above)
Comment (cjyetman) [reply R/lookup.R:19]: I think a unicode escape is always 4 characters, so no?
``` r
"1.5\u00B0C"
#> [1] "1.5°C"
```

Comment (jdhoffa) [reply R/lookup.R:19]: well nevuhmind!

**Review by cjyetman — CHANGES_REQUESTED** [review]
these test errors look bad enough to worry about

**Comment by MonikaFu** [comment]
the NOTES seem not to be related to the changes that I made. Should I look into this nevertheless? It has sth to do with 'emissions'

**Comment by MonikaFu** [comment]
@cjyetman checks are passing now

**Review by cjyetman — APPROVED** [review]

**Pull request merged**